src/nemos/solvers.py

```diff
@@ -14,12 +14,39 @@
 
 
 class SVRGState(NamedTuple):
+    """
+    Optimizer state for (Prox)SVRG.
+
+    Attributes
+    ----------
+    iter_num :
+        Current epoch or iteration number.
+    key :
+        Random key to use when sampling data points or mini-batches.
+    error :
+        Scaled difference (~distance) between subsequent parameter values
+        used to monitor convergence.
+    stepsize :
+        Step size of the individual gradient steps.
+    reference_point :
+        Anchor/reference/snapshot point where the full gradient is calculated in the SVRG algorithm.
+        Corresponds to `x_{s}` in the pseudocode in [1]
```

Comment:
@BalzaniEdoardo did you figure this out for mkdocs?

Reply:
I figured that our links for refs did not work because the References is not considered a markdown section. I did make it a section and this seems to be rendered well. Do you like this solution?

Reply:
I think this solution is fine for now, but we should figure out how to get them rendering correctly. Not as part of this PR, I opened an issue for it: #212